src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--5 .stars__star,
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+}
+
+// .stars--1 .stars__star:nth-child(n + 2),
+// .stars--2 .stars__star:nth-child(n + 3),
+// .stars--3 .stars__star:nth-child(n + 4),
+// .stars--4 .stars__star:nth-child(n + 5) {
+//   background-image: url(../images/star.svg);
+// }
```

Comment:
These commented lines of code should be removed if they are not used. It's not a good practice to leave commented out code in your final version, as it can cause confusion for other developers who might work on your code later.